application-forum-ui/src/main/resources/ForumCode/AnswerSheet.xml

```diff
@@ -53,6 +53,26 @@
         &lt;/dl&gt; 
       &lt;/div&gt;
     {{/html}}
+  #elseif ($request.commentEditForm == 'true')
+    {{html clean="false"}}
+      ## XWIKI-17318
+      #set ($commentEditForm = "#template('commentsinline.vm')")
+      #if (!$commentEditForm.contains('&lt;/textarea&gt;'))
+        ## XWiki 12.6+
```

Comment:
You need to be more explicit. Why isn't the ``textarea`` present starting with 12.6?